scripts/us_hud/income/README.md

```diff
@@ -15,4 +15,7 @@ The `match_bq.csv` file contains places that have additional dcids that we would
 To run unit tests: 
 ```
 python3 -m unittest discover -v -s ../ -p "*_test.py"
+
+you need to  install the required packages ,the python-calamine package for handling .xls files:
```

Comment:
pls reword to:
`Install the python-calamine package to process xls files by doing:
pip install python-calamine
`